test/storage/compression_test.cpp

```diff
@@ -0,0 +1,74 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// compression_test.cpp
+//
+// Identification: test/storage/compression_test.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "common/harness.h"
+
+#include "storage/data_table.h"
+
+#include "executor/testing_executor_util.h"
+#include "storage/tile_group.h"
+#include "storage/database.h"
+#include "storage/tile.h"
+#include "storage/tile_group_header.h"
+#include "storage/tuple.h"
+
+#include "concurrency/transaction_manager_factory.h"
+
+namespace peloton {
+namespace test {
+
+//===--------------------------------------------------------------------===//
+// Compression Tests
+//===--------------------------------------------------------------------===//
+
+std::unique_ptr<storage::DataTable> data_table_test_table;
+
+class CompressionTests : public PelotonTest {};
+
+TEST_F(CompressionTests, SizeTest) {
+  const int tuple_count = 100;
+  UNUSED_ATTRIBUTE oid_t tuple_size = 24;
+  UNUSED_ATTRIBUTE oid_t tuples_per_tile_group = 10;
+  auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+  auto txn = txn_manager.BeginTransaction();
+
+  data_table_test_table.reset(
+      TestingExecutorUtil::CreateTable(tuple_count, false));
```

Comment:
Check the definition of `TestingExecutorUtil::CreateTable` here:
https://github.com/cmu-db/peloton/blob/master/test/include/executor/testing_executor_util.h#L70
The first argument is actually `tuples_per_tile_group`.